prowler/providers/aws/services/waf/waf_service.py

```diff
@@ -14,15 +14,21 @@ def __init__(self, provider):
         self.rules = {}
         self.rule_groups = {}
         self.web_acls = {}
-        self._list_rules()
-        self.__threading_call__(self._get_rule, self.rules.values())
-        self._list_rule_groups()
-        self.__threading_call__(
-            self._list_activated_rules_in_rule_group, self.rule_groups.values()
-        )
-        self._list_web_acls()
-        self.__threading_call__(self._get_web_acl, self.web_acls.values())
-        self.__threading_call__(self._get_logging_configuration, self.web_acls.values())
+        if self.audited_partition == "aws":
```

Comment:
Why only `aws`? China has also CloudFront, at least by our regions file.

Reply:
You can see in https://docs.aws.amazon.com/general/latest/gr/waf-classic.html that only this partition is supported.